pythonforandroid/build.py

```diff
@@ -11,7 +11,7 @@
 import sh
 import subprocess
 
-from pythonforandroid.util import (ensure_dir, current_directory, BuildInterruptingException)
+from pythonforandroid.util import (current_directory, ensure_dir, get_virtualenv_executable, BuildInterruptingException)
```

Comment:
Pep8:
```
import (
    stuff
)
```

Reply:
I haven't enforced that one in tox yet (via `isort` + `E501`), but I do have it on my personal projects

Reply:
@KeyWeeUsr changed it! can you check if you actually like how it looks now?